[2024-12-06T18:39:38.173+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T18:39:38.196+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:39:36.916399+00:00 [queued]>
[2024-12-06T18:39:38.204+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:39:36.916399+00:00 [queued]>
[2024-12-06T18:39:38.205+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T18:39:38.216+0000] {taskinstance.py:2889} INFO - Executing <Task(SparkSubmitOperator): run_pyspark_etl> on 2024-12-06 18:39:36.916399+00:00
[2024-12-06T18:39:38.221+0000] {standard_task_runner.py:72} INFO - Started process 465 to run task
[2024-12-06T18:39:38.224+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pyspark_etl_dag', 'run_pyspark_etl', 'manual__2024-12-06T18:39:36.916399+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/scripts_etl_dags.py', '--cfg-path', '/tmp/tmp2omyhv5b']
[2024-12-06T18:39:38.227+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask run_pyspark_etl
[2024-12-06T18:39:38.249+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-06T18:39:38.292+0000] {task_command.py:467} INFO - Running <TaskInstance: pyspark_etl_dag.run_pyspark_etl manual__2024-12-06T18:39:36.916399+00:00 [running]> on host 9a0780602dc6
[2024-12-06T18:39:38.381+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pyspark_etl_dag' AIRFLOW_CTX_TASK_ID='run_pyspark_etl' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T18:39:36.916399+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T18:39:36.916399+00:00'
[2024-12-06T18:39:38.382+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T18:39:38.408+0000] {base.py:84} INFO - Retrieving connection 'spark_default'
[2024-12-06T18:39:38.409+0000] {spark_submit.py:473} INFO - Spark-Submit cmd: spark-submit --master yarn --conf spark.executor.memory=2g --conf spark.driver.memory=2g --name arrow-spark --queue root.default /opt/***/scripts/scripts_etl.py
[2024-12-06T18:39:40.430+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkContext: Running Spark version 3.4.4
[2024-12-06T18:39:40.470+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-12-06T18:39:40.532+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceUtils: ==============================================================
[2024-12-06T18:39:40.533+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-12-06T18:39:40.534+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceUtils: ==============================================================
[2024-12-06T18:39:40.535+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkContext: Submitted application: PostgreSQL Integration
[2024-12-06T18:39:40.549+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 2048, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-12-06T18:39:40.559+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-12-06T18:39:40.561+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-12-06T18:39:40.598+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SecurityManager: Changing view acls to: ***
[2024-12-06T18:39:40.599+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SecurityManager: Changing modify acls to: ***
[2024-12-06T18:39:40.600+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SecurityManager: Changing view acls groups to:
[2024-12-06T18:39:40.601+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SecurityManager: Changing modify acls groups to:
[2024-12-06T18:39:40.601+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-12-06T18:39:40.755+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO Utils: Successfully started service 'sparkDriver' on port 44907.
[2024-12-06T18:39:40.774+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkEnv: Registering MapOutputTracker
[2024-12-06T18:39:40.799+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkEnv: Registering BlockManagerMaster
[2024-12-06T18:39:40.812+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-12-06T18:39:40.813+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-12-06T18:39:40.839+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-12-06T18:39:40.854+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-1140adb0-9e4a-47d9-b74e-dd3aa9a1fbd1
[2024-12-06T18:39:40.867+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO MemoryStore: MemoryStore started with capacity 1048.8 MiB
[2024-12-06T18:39:40.894+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:40 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-12-06T18:39:41.000+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:41 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-12-06T18:39:41.046+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:41 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-12-06T18:39:41.073+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:41 INFO SparkContext: Added JAR /opt/***/scripts/postgresql-42.7.4.jar at spark://9a0780602dc6:44907/jars/postgresql-42.7.4.jar with timestamp 1733510380420
[2024-12-06T18:39:41.185+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:41 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-12-06T18:39:42.283+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:43.284+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:44.286+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:45.287+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:46.288+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:47.289+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:48.290+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:49.291+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:50.292+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:51.293+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:52.298+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:53.299+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:54.300+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:55.302+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:56.303+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:57.304+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:58.305+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:39:59.307+0000] {spark_submit.py:634} INFO - 24/12/06 18:39:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:00.307+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:01.308+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:01.309+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:01 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 32960ms.
[2024-12-06T18:40:35.269+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:36.271+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:37.272+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:38.273+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:39.274+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:40.276+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:41.277+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:42.277+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:43.278+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:44.279+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:40:44.280+0000] {spark_submit.py:634} INFO - 24/12/06 18:40:44 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 21024ms.
[2024-12-06T18:41:06.304+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:07.304+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:08.305+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:09.306+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:10.307+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:11.308+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:12.309+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:13.310+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:14.311+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:15.312+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:15.313+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:15 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 26456ms.
[2024-12-06T18:41:42.771+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:42 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:43.773+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:43 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:46.327+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:44 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:46.329+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:46.777+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:47.650+0000] {job.py:229} INFO - Heartbeat recovered after 12.69 seconds
[2024-12-06T18:41:47.778+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:48.779+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:49.780+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:50.781+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:51.782+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:41:51.783+0000] {spark_submit.py:634} INFO - 24/12/06 18:41:51 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 33772ms.
[2024-12-06T18:41:53.511+0000] {job.py:229} INFO - Heartbeat recovered after 13.03 seconds
[2024-12-06T18:42:26.554+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:26 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:27.555+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:27 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:28.556+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:28 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:29.557+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:29 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:30.555+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:30 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:31.555+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:31 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:32.556+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:33.557+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:34.558+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:35.559+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:42:35.560+0000] {spark_submit.py:634} INFO - 24/12/06 18:42:35 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 34051ms.
[2024-12-06T18:42:58.261+0000] {job.py:229} INFO - Heartbeat recovered after 13.43 seconds
[2024-12-06T18:43:03.365+0000] {job.py:229} INFO - Heartbeat recovered after 11.96 seconds
[2024-12-06T18:43:10.612+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:11.614+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:12.614+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:13.615+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:14.616+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:15.617+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:16.618+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:17.620+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:18.621+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:19.375+0000] {job.py:229} INFO - Heartbeat recovered after 10.93 seconds
[2024-12-06T18:43:19.622+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-12-06T18:43:20.180+0000] {spark_submit.py:634} INFO - 24/12/06 18:43:19 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 6 failover attempts. Trying to failover after sleeping for 33297ms.
